test/integration-test.sh

```diff
@@ -2,7 +2,9 @@
 set -ex
 
 DIR=$PWD
-go build ./cmd/stupid
+#go build ./cmd/stupid
```

Comment:
if we don't need them, pls kill these lines. otherwise, pls comment on why we need to keep them